### PR TITLE
add docs for fixed packages

### PR DESCRIPTION
**Pull request by akphi**
Closes https://github.com/changesets/changesets/issues/754

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ To make releasing easier, you can use [this changesets github action](https://gi
 - [Decisions](./docs/decisions.md)
 - [Dictionary](./docs/dictionary.md)
 - [Linked packages](./docs/linked-packages.md)
+- [Fixed packages](./docs/fixed-packages.md)
```
Comment (Andarist) [inline README.md:53]: from my experience people were often confused about linked packages - they actually assumed that they would work like fixed ones, so perhaps we should reorder those here to prioritize fixed over linked? the same comment goes for other places were you put fixed below linked within this PR

File: docs/config-file-options.md [hunk 2]
```diff
@@ -65,6 +65,18 @@ If you want to use this option, you should read the documentation on [linked pac
 
 > NOTE: This does not do what some other tools do, which is make sure when any package is published, all other packages are also published with the same version.
 
+## `fixed` (array of arrays of package names)
+
+This option can be used to declare that packages should be version-bumped and published together. As an example, if you have a `@changesets/button` component and a `@changesets/theme` component and you want to make sure that when one gets bumped to `2.0.0`, the other is also bumped to `2.0.0` regardless if it has any change or not. To achieve this you would have the config:
```
Comment (Andarist) [inline docs/config-file-options.md:70]: It feels that using a major version example could be confusing here cause it's less common situation and somebody could think that the behavior only applies to major versions cause of that.
```suggestion
This option can be used to declare that packages should be version-bumped and published together. As an example, if you have a `@changesets/button` component and a `@changesets/theme` component and you want to make sure that when one gets bumped to `1.1.0`, the other is also bumped to `1.1.0` regardless if it has any change or not. To achieve this you would have the config:
```

File: docs/dictionary.md [hunk 3]
```diff
@@ -17,6 +17,7 @@ Several of these have associated type definitions, which you can find in [our ty
 - **multi-package repo/monorepo** - A repository which contains multiple packages, generally managed by [Bolt](https://github.com/boltpkg/bolt) or [Yarn Workspaces](https://yarnpkg.com/lang/en/docs/workspaces/).
 - **release line generators** - The `getReleaseLine` and `getDependencyReleaseLine` functions which are responsible for creating the lines inserted into changelog. A changelog entry for a particular release can be thought of as `releaseLineGenerators(changesets)`
 - **linked packages** - Linked packages share a semver categorisation, such that all published linked packages have consistent new semver ranges. The logistics of this are best left to our [./linked-packages.md] documentation.
+- **fixed packages** - Fixed packages share a semver categorisation, such that all linked packages have consistent new semver ranges and will be published together. The logistics of this are best left to our [./fixed-packages.md] documentation.
```
Comment (Andarist) [inline docs/dictionary.md:20]:
```suggestion
- **fixed packages** - Fixed packages share a semver categorisation, such that all fixed packages have the same semver version and are always published together. The logistics of this are best left to our [./fixed-packages.md] documentation.
```

File: docs/fixed-packages.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# Fixed Packages
+
+Fixed packages allow you to specify a group or groups of packages that should be versioned and published together.
+
+> Unlike `linked packages`, all packages in the group of fixed packages will be version-bumped and published together even when there are no changes done to some of the the member packages.
```
Comment (Andarist) [inline docs/fixed-packages.md:5]: a similar note (just "in reverse") could be added to the linked packages doc

**Comment by Andarist** [comment]
❤️ ❤️ ❤️

**Comment by akphi** [comment]
Updated!

File: docs/fixed-packages.md (hunk 4 above)
Comment (akphi) [reply docs/fixed-packages.md:5]: updated!

File: README.md (hunk 1 above)
Comment (akphi) [reply README.md]: done

**Review by Andarist — APPROVED** [review]

**Pull request merged**